dqc_us_rules/_pkg_meta.py

```diff
@@ -0,0 +1,5 @@
+"""
+Module Docstring
```

Comment:
This isn't a very informative docstring... is it necessary? or is there a better description?